src/main/java/org/opendatakit/aggregate/servlet/ServiceAccountPrivateKeyUploadServlet.java

```diff
@@ -247,6 +247,7 @@ public void doPost(HttpServletRequest req, HttpServletResponse resp) throws IOEx
         out.write(HtmlConsts.HTML_OPEN);
         out.write(HtmlConsts.BODY_OPEN);
         out.write("<p>Successful private key information upload.</p>");
+        out.write("<p><strong>Note</strong>: If you have updated the Google Simple API Key, please, reload this page pressing the F5 key.</p>");
```

Comment:
People don't know about F5 and we can't guarantee the browser so keyboard shortcuts are less helpful. 

> If you have updated the Google Simple API Key, please reload/refresh this page.